app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,27 @@
   </head>
 
   <body>
+    <%= render "partials/navbar" %>
+
+    <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
```

Comment:
You should only code on application.html.erb those things which will show up in **every** page. That is the case of the navbar, but not the case of the banner (assuming that this carousel will be a banner and not the background (in which case it should not be a carousel). 
Please, put this in app/views/pages/home.html.erb
Furthermore, you should use a partial, just as you used for the navbar. So, instead of putting this directly in home.html.erb, please create a file _banner.html.erb in partials and render it in home.html.erb.